jmx-metrics/docs/target-systems/solr.md

```diff
@@ -0,0 +1,84 @@
+# Solr Metrics
+
+The JMX Metric Gatherer provides built in Solr metric gathering capabilities.
```

Comment:
Do you know of a jmx monitoring reference in the solr docs that can be linked to?

Reply:
I'm not versed in solr but https://solr.apache.org/guide/6_6/using-jmx-with-solr.html seems like a fine launching point to show it's a "blessed" monitoring interface

Reply:
@rmfitzpatrick I can throw in a link to it.